company_service/urls.py

```diff
@@ -32,7 +33,10 @@
 code_group_router.register('rework_codes', views.ReworkCodeViewSet, base_name='companies-code_groups-rework_codes')
 
 production_order_router = routers.NestedSimpleRouter(companies_router, 'production_orders', lookup='production_orders')
-production_order_router.register('production_events', views.ProductionEventViewSet, base_name='companies-production_orders-production_events')
+production_order_router.register('production_events', views.ProductionEventViewSet, basename='companies-production_orders-production_events')
```

Comment:
base_name ou basename?

```suggestion
production_order_router.register('production_events', views.ProductionEventViewSet, base_name='companies-production_orders-production_events')
```